src/Enterspeed.Source.UmbracoCms.Base/Providers/EnterspeedHttpContextProvider.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace Enterspeed.Source.UmbracoCms.Base.Providers
+{
+    public class EnterspeedHttpContextProvider : IEnterspeedHttpContextProvider
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly IServiceProvider _serviceProvider;
+
+        public EnterspeedHttpContextProvider(
+            IHttpContextAccessor httpContextAccessor,
+            IServiceProvider serviceProvider)
+        {
+            _httpContextAccessor = httpContextAccessor;
+            _serviceProvider = serviceProvider;
+        }
+
+        public IDisposable CreateFakeHttpContext()
+        {
+            var httpContext = new DefaultHttpContext
+            {
+                Request =
+                {
+                    Scheme = "https",
+                    Host = new HostString("enterspeed.com")
```

Comment:
Er dette blot et dummy hostname eller bruges denne httpContext til noget?

Reply:
Det er et dummy hostname, for at få en valid context. Umbraco bruger ikke hostname til noget i koden der bliver eksekverede i baggrunden.